vlsv_reader_parallel.cpp

```diff
@@ -430,10 +430,13 @@ namespace vlsv {
       // Attempt to open the given input file using MPI:
       fileName = fname;
       int accessMode = MPI_MODE_RDONLY;
-      if (MPI_File_open(comm,const_cast<char*>(fileName.c_str()),accessMode,mpiInfo,&filePtr) != MPI_SUCCESS) success = false;
-      else parallelFileOpen = true;
-      
-      if (success == false) cerr << "Failed to open parallel file" << endl;
+      int err {MPI_File_open(comm,const_cast<char*>(fileName.c_str()),accessMode,mpiInfo,&filePtr)};
```

Comment:
"While valid, this is stupid." :) Initializers do have their uses but let's be realistic, here it's just odd. (TBH is odd also on line 420 since it's just a single variable)